github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -323,3 +323,44 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
```

Comment:
can we use `run_if_changed` to narrow down when there are changes on the image setup?

Reply:
Then rehearse will not test this, I think is not worth it.

Reply:
OK, as it uses less resources now, it should be fine.